iconrpcserver/dispatcher/default/node.py

```diff
@@ -89,3 +86,13 @@ class NodeDispatcher:
         block_hash, response = await get_block_by_params(channel_name=channel, block_height=request['height'],
                                                          with_commit_state=True)
         return response
+
+    @staticmethod
+    @methods.add
+    async def node_getCitizens(**kwargs):
+        channel = kwargs['context']['channel']
+        channel_stub = get_channel_stub_by_channel_name(channel)
+        citizens: list = await channel_stub.async_task().get_citizens()
```

Comment:
This would be better update to `get_citizens()` return type. [here](https://github.com/icon-project/icon-rpc-server/pull/61/files#diff-692147a4414f06f7ae3779af7a25ee8eR59)